kevm

```diff
@@ -183,6 +190,7 @@ if [[ "$run_command" == 'help' ]] || [[ "$run_command" == '--help' ]] ; then
                $0 kast         [--backend (llvm|java|haskell)]           <KEVM_arg>* <pgm>  <output format> <K arg>*
                $0 prove        [--backend (java|haskell)]                            <spec> <def_module> <K arg>*
                $0 search       [--backend (java|haskell)]                            <pgm>  <pattern> <K arg>*
+               $0 kompile      [--backend (llvm|haskell)]                            <main> <K arg>*
```

Comment:
```suggestion
               $0 kompile      [--backend (java|llvm|haskell)]                       <main> <K arg>*
```

It looks like it should be like this?

Reply:
Sure, I'm just in general trying not to spread use of the java backend, but makes sense.